.github/workflows/php-cs-fixer.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           ref: ${{ github.head_ref }}
 
       - name: Run PHP CS Fixer
-        uses: docker://oskarstark/php-cs-fixer-ga
+        uses: docker://oskarstark/php-cs-fixer-ga:2.18.6
```

Comment:
@joshtorres 

changing the name 
https://github.com/tipoff/notes/pull/41/commits/fa8edf397d951582abb8d1f45061286148afa866  

I got 
https://github.com/tipoff/notes/runs/2505245000

but with this approach works